pkg/util/node/node.go

```diff
@@ -187,16 +187,17 @@ func GetZoneKey(node *v1.Node) string {
 		return ""
 	}
 
-	// TODO: prefer stable labels for zone in v1.18
-	zone, ok := labels[v1.LabelZoneFailureDomain]
+	// TODO: "failure-domain.beta..." names are deprecated, but will
+	// stick around a long time due to existing on old extant objects like PVs.
+	// Maybe one day we can stop considering them (see #88493).
+	zone, ok := labels[v1.LabelFailureDomainBetaZone]
```

Comment:
Would it be safe at this point to at least switch the priority order here so that the topology labels are checked first?

Reply:
Same for regions below

Reply:
On second thought, it might be cleaner to keep this PR scoped to the new constants and do the priority switch in a follow-up PR

Reply:
If at all?  I am not sure it makes any real sense to switch the priority.  If the labels are the same, no-op.  If they are different, best case is that we break users.